### PR TITLE
1st Capstone Project

### PR DESCRIPTION
**Pull request by vigehi**
This is my final capstone project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+- Git and Github
+
+
+## Authors
+
+👤 **Edith Oga**
+
+- GitHub: [@vigehi](https://github.com/vigehi)
+
+[View Live](https://vigehi.github.io/project1/)
```
Comment (ShoiraTa) [inline README.md:34]: - [x] Kudos on properly deploying the project!

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+
+## Loom video
+[View loom video](https://www.loom.com/share/a3851b9e7d7443d69f3722ea30bda7ee)
+
```
Comment (ShoiraTa) [inline README.md]: - [x] Please check the video description, it does not have audio.

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+
+## Built With
+
+- HTML
+- CSS
+- Bootsrap
+- JavaScript
+- Git and Github
+
```
Comment (ShoiraTa) [inline README.md:14]: - [x] Kindly add getting started steps, where you describe how users can locally install your project and view on browser. There are many different languages, and technologies, all of them require various steps to get started. Perhaps for this project it is enough just to clone the project and run` npm install` to install dependencies, but it is a good habit to write getting started steps into your documentation as your projects will get more complicated 😉

File: README.md [hunk 4]
```diff
@@ -1 +1,39 @@
-# html-capstone-project1
\ No newline at end of file
+# blessed-adegu sessions
+
+> This is my first Microverse capstone project.
+
```
Comment (ShoiraTa) [inline README.md]: - [x] Kindly add a proper pull request description. Use your PR description as an opportunity to set the reviewer up for a review they will want to do, a review they will know how to review. You should describe what you have done implemented in this Branch. 
Currently it is just `This is my final capstone project`, as you can guess by now, this is not descriptive. Of course I know what had to be done in this project, but again, writing good Pull requests will help you to generate a habit of following good dev habits !

File: about.html [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+              class="collapse navbar-collapse inside-desktop-nav"
+              id="navbarNav"
+            >
+              <ul class="navbar-nav">
+                <li class="nav-item">
+                  <a class="nav-link" href="./about.html">About</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Program</a>
+                </li>
```
Comment (ShoiraTa) [inline about.html]: - [x] Kindly add a link to Program page, like you did for logo 😉

File: about.html [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+        <div class="col-sm-6 col-md-2 top-program-2b">
+          <h4>2020</h4>
+          <p>Harmony SDA Church sessions</p>
+        </div>
+        <div class="col-sm-6 col-md-2 top-program-2b">
+          <h4>2021</h4>
+          <p>Namunyiri SDA church sessions</p>
+        </div>
+      </div>
+    </section>
```
Comment (ShoiraTa) [inline about.html]: - [x] Kindly add images in cards and more space below cards, as provided in design:

Your design  | Behance design
------------- | -------------
Screenshot 2022-08-04 at 09 55 07 | Screenshot 2022-08-04 at 09 54 48

File: about.html [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+    <footer>
+      <div class="footer-img">
+        <img class="footer-img" src="./images/logo.png" alt="logo" />
+      </div>
+      <div class="footer-text">
+        <p>@2022 Blessed adegu talk</p>
+      </div>
+    </footer>
```
Comment (ShoiraTa) [inline about.html:225]: - [x] In About page, footer should be: 
    - Desktop Size:  It has a dark background.
    - Mobile Size:  It has a white background. 
  Kindly hide sponsors section in desktop size only for About page:

Your design  | Behance design
------------- | -------------
Screenshot 2022-08-04 at 09 59 17  | Screenshot 2022-08-04 at 09 59 03

File: about.html [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+      </div>
+    </section>
+
+    <section class="patners">
+      <div class="row outer-bottom-partner-head">
+        <div class="col-md-6 bottom-partner-head">
+          <h2>Our Sponsors</h2>
+          <hr />
+        </div>
+      </div>
```
Comment (ShoiraTa) [inline about.html:194]: - [x] Kindly horizontally center logos in `Our Sponsors section`. Now they are aligned to the left. 
Screenshot 2022-08-04 at 10 00 59
 

- [x] Kindly use colors as provided in Design, the `Sponsors section` is black, but should be #271A31 color
Screenshot 2022-08-04 at 10 02 01

- [x] In mobile size,` Sponsors section` is breaking, kindly fix it

Screenshot 2022-08-04 at 10 03 42

File: about.html [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+            </div>
+            <div class="col col-md-2 foot">
+              <img class="footer-img" src="./images/partner/gc.png" alt="" />
+            </div>
+            <div class="col col-md-2 foota">
+              <img
+                class="footer-img"
+                src="./images/partner/images.png"
+                alt=""
+              />
```
Comment (ShoiraTa) [inline about.html:212]: - [x] Make sure to have alt descriptions for images where necessary.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @vigehi ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- You passed all the Linters! ✔️ 
- Project is properly deployed ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [x] Please try to write meaningful commit messages, I can see many repeated commits now, like updates repeated 3 times, update on CSS repeated 4 times. 

- [x] _Check more the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (vigehi) [reply README.md:34]: Thank you

File: README.md (hunk 3 above)
Comment (vigehi) [reply README.md:14]: okay

File: README.md (hunk 4 above)
Comment (vigehi) [reply README.md]: Thank you

File: about.html (hunk 5 above)
Comment (vigehi) [reply about.html]: Thank you for the review

File: about.html (hunk 6 above)
Comment (vigehi) [reply about.html]: thank you for the review

File: README.md (hunk 4 above)
Comment (Lordkaito) [reply README.md]: - [ ] As mentioned by the previous reviewer, please make sure to add a descriptive pull request. Think of a pull request as a text that should give a quick idea of what you did in these changes, summarizing the general content.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @vigehi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (vigehi) [reply README.md]: i have resolved this

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @vigehi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**